src/NuGet.Clients/NuGet.CommandLine/Common/Console.cs

```diff
@@ -252,23 +263,30 @@ public void PrintJustified(int startIndex, string text, int maxWidth)
         {
             if (maxWidth > startIndex)
             {
-                maxWidth = maxWidth - startIndex - 1;
+                maxWidth = maxWidth - startIndex;
             }
 
             lock (_writerLock)
             {
                 while (text.Length > 0)
                 {
                     // Trim whitespace at the beginning
-                    text = text.TrimStart();
+                    text = text.TrimStart(' ', '\t');
```

Comment:
Is this necessary? I thought Trim already removed tab?

Reply:
TrimStart() was removing preceding newlines, which prevented the algorithm from correctly determining the number of newlines that needed to be added.